### PR TITLE
Feature nxp 21386 split multi tenant in core and jsf

### PR DESCRIPTION
**Pull request by egiuly-nuxeo**
https://jira.nuxeo.com/browse/NXP-21386

### CODE REVIEW

**Comment by efge** [comment]
👍

File: nuxeo-multi-tenant-core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-login</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-web-common</artifactId>
+      <scope>test</scope>
+    </dependency>
```
Comment (atchertchian) [inline nuxeo-multi-tenant-core/pom.xml]: please check dependencies as these are probably not needed on the core part (mvn dependency:analyze is your friend)

File: nuxeo-multi-tenant-jsf/pom.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <groupId>org.nuxeo.multi.tenant</groupId>
+  <artifactId>nuxeo-multi-tenant-jsf</artifactId>
+  <name>Nuxeo Multi Tenant JSF</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.nuxeo.multi.tenant</groupId>
+      <artifactId>nuxeo-multi-tenant-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (atchertchian) [inline nuxeo-multi-tenant-jsf/pom.xml]: project.version should not be needed here since you added the core module to the parent pom dependency management

Comment (egiuly-nuxeo) [inline nuxeo-multi-tenant-jsf/pom.xml]: Ok, done!

File: nuxeo-multi-tenant-core/pom.xml (hunk 1 above)
Comment (jcarsique) [inline nuxeo-multi-tenant-core/pom.xml]: :+1: See https://doc.nuxeo.com/corg/maven-coding-rules/#direct-versus-transitive-dependencies
(and format the file in the mean time)

**Pull request merged**